User.js

```diff
@@ -0,0 +1,30 @@
+//The User class can be utililized to keep certain information of the user, a class for the user needed to be 
+// created in addition to the session class. The test for this JS class file utilizes Jest.IO
+
+class User {
+    constructor() {
+        const expected = {
+            sessionId: 'JTC6',
+            controllerStatus = true,
+            userIp = "129.23.321.12"
+        };
```

Comment:
Same thing here, create fromObject(obj). Lets avoid random placeholders are much as possible and try to leverage good API design to bypass random placeholders